code/modules/clothing/head/jobs.dm

```diff
@@ -194,22 +196,39 @@
 
 	create_storage(storage_type = /datum/storage/pockets/small/fedora/detective)
 
+	RegisterSignal(src, COMSIG_CLICK_ALT, PROC_REF(on_clicked_alt))
```

Comment:
Can we disable the alt-click hotkey for storage rather than needing to do this?

Reply:
Or maybe make it alt-right-click I guess

Reply:
i tried building something that fits this requirement, let me know if it works